File: Array/Linear Search/linear_search.java
//Linear search
import java.util.Scanner;
public class linear_search {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the size of array");
        int n = sc.nextInt();
        int arr[] = new int[n];
        System.out.println("Enter the elements of array");
        for (int i = 0; i < n; i++) {
            arr[i] = sc.nextInt();
        }
        System.out.println("Enter the element to be searched");
        int key = sc.nextInt();
        int flag = 0;
        for (int i = 0; i < n; i++) {
            if (arr[i] == key) {
                System.out.println("Element found at position " + (i + 1));
                flag = 1;
                break;
            }
        }
        if (flag == 0) {
            System.out.println("Element not found");
        }
    }
}

File: Searching/Linear Search/linear_search.py
# linear search
def search(arr, x):
 
    for i in range(len(arr)):
 
        if arr[i] == x:
            return i
 
    return -1

l = input("Enter the list of numbers: ").split()
print("The list of numbers is: ", search(
    l, input("Enter the number to search: ")))
